mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,51 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+     b is undefined
   2. Remove the line that throws the error.
+     line 35
   3. What is printed to the console?
+      2
+      6
+      4
+      9
+      6
+      13
+      8
   4. How many times is "f1" called?
+     3times
   5. How many times is "f2" called?
+     2times
   6. What value does the "a" parameter take in the first "f1" call?
+     8
```

Comment:
This is a confusing question, but it means what is the value of a in the line 
`const f1 = function (a, b)`
when it is called for the first time. I can see from your commented out code below that you do know what this is.